src/default-options.ts

```diff
@@ -26,6 +26,12 @@ export const defaultOptions = {
     // in bytes, with the default lease manager.
     maxOutstandingBytes: 100 * 1024 * 1024,
 
+    // The minimum length of time a message's lease will be extended by.
+    minAckDeadline: undefined,
+
+    // The maximum length of time a message's lease will be extended by.
+    maxAckDeadline: undefined,
```

Comment:
Similar to above, the Java library has the `maxAckDeadline` default to 600 seconds

Reply:
For Java, we keep track if user-values are used for the deadlines, and if they are not, we change to use the EOS 10/600 defaults.